pkgs/applications/audio/midisheetmusic/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchurl
+, mono, dotnetPackages, makeWrapper
+, gtk2, cups, timidity }:
+
+let
+  version = "2.6";
+in stdenv.mkDerivation {
+  name = "midisheetmusic";
```

Comment:
Dang, I just noticed version is not part of name, that should be fixed.

Reply:
@joachifm Thanks! https://github.com/NixOS/nixpkgs/pull/48109